File: Core Java/accountPolymorphism/src/com/techlabs/currentAccount/CurrentAccount.java
package com.techlabs.currentAccount;

import com.techlabs.account.Account;

public class CurrentAccount extends Account {

	public CurrentAccount(int account, String name, double balance) {
		super(account, name, balance);
	}

	@Override
	public void withdraw(double balance) {

		if (this.balance >= 5000) {
			this.balance = this.balance - balance;
		} else {
			throw new RuntimeException(
					"Your balance is low Minimum balance require 500");
		}
	}

}

File: DOTNET/C#/AccountPolymorphism/AccountPolymorphism/CurrentAccount.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AccountPolymorphism
{
    class CurrentAccount : Account
    {
        public CurrentAccount(int accountNumber, string name, double balance) : base(accountNumber, name, balance)
        {
        }

        public override void WithDraw(double balance)
        {
            if (this.Balance >= 5000)
            {
                this.balance = this.balance - balance;
            }
            else
            {
                throw new Exception("Your Balance is low.Minimum balance require is 500..");
            }
        }
    }
}
